man/roxylint/meta.R

```diff
@@ -0,0 +1,13 @@
+list(
+  linters = list(
+    typed = list(
+      "descriptions should start lowercase" = "^[^[:upper:]]",
+      roxylint::lint_full_stop,
+      NULL  # disallow registration of package defaults
+    ),
```

Comment:
Defaults are overwritten. The `NULL` at the end signals that no new rules should be added. Without it, new default rules for the `@typed` tag are registered when `roxytypes` is loaded. 

What's important is that the config needs to allow users to choose whether to allow or disallow new rules. That said, ending with `NULL` as a meaningful flag still feels clunky to me.